forms-flow-web/src/components/Form/List.js

```diff
@@ -223,7 +223,7 @@ const List = React.memo((props) => {
       updatedQuery = `${isAscend ? "-" : ""}title`;
       dispatch(setBPMFormListSort(updatedQuery));
     } else {
-      const updatedQuery = isAscend ? 'asc' : "desc";
+      const updatedQuery = isAscend ? 'desc' : "asc";
```

Comment:
can you verify the changes, I think logically the code is the same as before. I don't think this will solve anything.

Reply:
The issue was when the form is rendered, and on the first click on the sort was not making any changes since the default sort is asc. now the default will trigger at the first click itself and this will be chaged.

Reply:
so according to the changes the default sort order will be desc, correct? is that what we need? if so change the variable names appropriately for better readability.